### PR TITLE
switch to go modules

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

**Comment by tariq1890** [comment]
@databus23 FYI

**Comment by tariq1890** [comment]
@databus23 Friendly ping :)

**Comment by tariq1890** [comment]
Requesting your review @databus23

**Comment by deiga** [comment]
Maybe @mumoshu can help here as well? :)

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-HAS_GLIDE := $(shell command -v glide;)
 VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 
 PKG:= github.com/databus23/helm-diff
 LDFLAGS := -X $(PKG)/cmd.Version=$(VERSION)
 
 # Clear the "unreleased" string in BuildMetadata
 LDFLAGS += -X $(PKG)/vendor/k8s.io/helm/pkg/version.BuildMetadata=
-LDFLAGS += -X $(PKG)/vendor/k8s.io/helm/pkg/version.Version=$(shell grep -A1 "package: k8s.io/helm" glide.yaml | sed -n -e 's/[ ]*version:.*\(v[.0-9]*\).*/\1/p')
+LDFLAGS += -X $(PKG)/vendor/k8s.io/helm/pkg/version.Version=$(shell grep -A1 "package: k8s.io/helm" go.mod | sed -n -e 's/[ ]*version:.*\(v[.0-9]*\).*/\1/p')
```
Comment (mumoshu) [inline Makefile:9]: I think this should now grep for `k8s.io/helm v2.11.0+incompatible` in `go.mod` and the sed expr should be rewritten to extract `v2.11.0` only. But let's defer it to another issue/pr as I've left this not reviewed too long!

**Pull request merged**